containers/ddev-webserver/ddev-webserver-base-files/etc/bashrc/commandline-addons.bashrc

```diff
@@ -1,4 +1,4 @@
-export PATH="~/bin:$PATH:/var/www/html/vendor/bin:/var/www/html/bin:$DDEV_COMPOSER_ROOT/vendor/bin"
```

Comment:
I see `/var/www/html/bin` removed here, that was added in

- #2949

Reply:
Oh, that was a mistake. I was just trying to remove ~/bin.

Reply:
I added it back in.